src/main/java/org/openjdk/jextract/impl/HeaderFileBuilder.java

```diff
@@ -558,32 +569,45 @@ private void emitConstant(Class<?> javaType, String constantName, Object value,
         incrAlign();
         if (value instanceof String) {
             emitDocComment(declaration);
-            appendLines(STR."""
-                public static \{javaType.getSimpleName()} \{constantName}() {
-                    class Holder {
-                        static final \{javaType.getSimpleName()} \{constantName}
-                            = \{runtimeHelperName()}.LIBRARY_ARENA.allocateFrom("\{Utils.quote(Objects.toString(value))}");
-                    }
-                    return Holder.\{constantName};
-                }
-                """);
+            appendLines(String.format("""
+        public static %s %s() {
```

Comment:
Why the negative indentation here?

Reply:
Fixed in [6adcab3](https://github.com/openjdk/jextract/pull/244/commits/6adcab3e23db3e03ee08a05889e23de31d088d2d), I think I didn't see them in the diff.